checkup.go

```diff
@@ -359,6 +359,12 @@ type Storage interface {
 	Store([]Result) error
 }
 
+// StorageReader can read results from the Storage.
+type StorageReader interface {
+	Fetch(string) ([]Result, error)
+	GetIndex() (map[string]int64, error)
```

Comment:
This interface is hard to understand. What is the `string` parameter in `Fetch`? What is the map of `strings` to `int64` being returned by `GetIndex`?

It'll be hard to implement this interface without knowing what it's expected to do.

A fix is to document the methods and give parameter names, e.g., `Fetch(name string) ([]Result, error)` is more readable than `Fetch(string) ([]Result, error)`.

See https://godoc.org/net/http#CookieJar for an example of a well documented interface.